rustc_codegen_spirv/src/builder/intrinsics.rs

```diff
@@ -240,7 +241,33 @@ impl<'a, 'tcx> IntrinsicCallMethods<'tcx> for Builder<'a, 'tcx> {
                 if self.kernel_mode {
                     self.cl_op(CLOp::copysign, [args[0].immediate(), args[1].immediate()])
                 } else {
-                    self.fatal("TODO: Shader copysign not supported yet https://github.com/EmbarkStudios/rust-gpu/issues/148")
+                    let arg0 = args[0].immediate();
+                    let arg1 = args[1].immediate();
```

Comment:
I think it would be more clear if you use `sign`, `sign_val`, `sign_src` or something like that instead of `arg1`.